flink-metric-prometheus/README.md

```diff
@@ -0,0 +1,82 @@
+# Overview
+
+This example shows how to report metrics defined in a FeatureView to a
+Prometheus PushGateway. It involves the following steps:
+
+1. Read a batch of historical purchase events from a file.
```

Comment:
Can we also use Kafka in this example?

Feature Stores such as Tecton also supports feature metrics in batch mode. The ability to support real-time feature metrics differentiates FeatHub from these systems.